.gitignore

```diff
@@ -7,4 +7,5 @@ venv/
 /outline-install.log
 __pycache__
 .coverage
-coverage.xml
\ No newline at end of file
+coverage.xml
+*.env
```

Comment:
Please add a new line at the end of the line here. It was also not in the original file but it should be.